packages/golden_toolkit/lib/src/multi_screen_golden.dart

```diff
@@ -55,16 +55,13 @@ Future<void> multiScreenGolden(
   double overrideGoldenHeight,
   CustomPump customPump,
   DeviceSetup deviceSetup,
-  List<Device> devices = const [
-    Device.phone,
-    Device.tabletLandscape,
-  ],
+  List<Device> devices,
   @Deprecated('This method level parameter will be removed in an upcoming release. This can be configured globally. If you have concerns, please file an issue with your use case.')
       bool skip,
 }) async {
-  assert(devices?.isNotEmpty ?? false);
+  assert(devices == null || devices.isNotEmpty);
```

Comment:
this parameter can now be null, in which we case we fall back to the global config.